recipes/segregation/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 13 is convenient
+# when copying and pasting from another recipe, but not really needed.
+# The sha256 is retrieved from the .tar.gz in the PyPI
+{% set name = "segregation" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pysal/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1f5a694936799accc977538dc2d31fcd434272dd40c77102599ed808239006c5
+ 
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - pandas
+    - geopandas
+    - matplotlib
+    - scikit-learn
+    - seaborn
+    - numpy
+    - scipy
+    - libpysal
+
+test:
+  imports:
+    - segregation
+
+about:
+  home: https://pysal.org
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Segregation Measures Framework in PySAL'
+
+  description: |
+    The PySAL segregation module allow users to estimate several segregation measures and perform 
+    inference for single measures and comparative inference in a concise way.
+  doc_url: http://segregation.readthedocs.io/
+  dev_url: https://github.com/pysal/segregation
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```

Comment:
Can you remove this comment from the example

Reply:
Of course, just solved this in the last commit! Please let me know if anything else is needed!